docs/source/install_scripted.rst

```diff
@@ -38,30 +38,21 @@ The output appears similar to the following example::
 
 Installing without Internet Connectivity
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If you choose the offline option to run the installer, the following associated dependencies must be installed.
+If you choose the offline option to run the installer, you will need to obtain the following packages on a machine with internet connectivity. Most of the packages can be obtained through the `Github Release Page <https://github.com/quipucords/quipucords/releases/tag/0.0.46>`_.
 
-**Server Dependencies:**
-
-- `Ansible <install.html#installing-the-ansible-prerequisite>`_
-- `Docker <install.html#installing-docker-and-the-quipucords-server-container-image>`_
+**Server Package:**
 
-**Command Line Tool Dependencies:**
-
-- `Download & Configure EPEL <install.html#commandline>`_
-- Python 3.4
-- Python 3.4-requests
+- Server Container Image (`Download <https://github.com/quipucords/quipucords/releases/download/0.0.46/quipucords.0.0.46.tar.gz>`_)
 
-Packages
-++++++++
-After the dependencies are installed, you must gather packages through the `Github Release Page <https://github.com/quipucords/quipucords/releases/tag/0.0.46>`_.
+**Build the Postgres Image:**
 
-**Server Packages:**
+- Install Docker (`Documentation <https://docs.docker.com/install/>`_)
```

Comment:
Let's put a little introduction here.

```
The Quipucords server requires a Postgres Docker image.  You must build the image on a machine with internet connectivity.  Follow the steps documented below to create a Postgres Docker image.
```

Ashley will probably correct that more, but that is the general idea.